{"uuid": "922f2bb3-a9b0-462f-a96d-508e23d1867c", "children": ["7e2ce684-7840-4d9e-a4bd-57abcb456665"], "befores": [{"name": "action", "status": "broken", "statusDetails": {"message": "AttributeError: module 'allure' has no attribute 'environment'\n", "trace": "  File \"/usr/local/Cellar/python/3.6.5/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/Cellar/python/3.6.5/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/_pytest/fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/Cellar/python/3.6.5/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/_pytest/fixtures.py\", line 792, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"/Users/test/work/API_Automation/TestCase/conftest.py\", line 35, in action\n    allure.environment(environment=env)\n"}, "start": 1586763489474, "stop": 1586763489475}], "start": 1586763489474, "stop": 1586763489576}